exercise/models.py

```diff
@@ -189,7 +189,7 @@ def __str__(self):
 
 class Exercise(PhishtrayBaseModel):
     def __str__(self):
-        return self.title
+        return f"{self.id} - {self.title}"
```

Comment:
I would swap these around since `title` is going to be more readable/recognisable.
```suggestion
        return f"{self.title} - {self.id}"
```